pyproject.toml

```diff
@@ -0,0 +1,23 @@
+[tool.poetry]
+name = "mbta-covid-recovery-dash"
+version = "0.1.0"
+description = "Tracking changes in MBTA service levels and ridership during the Covid-19 pandemic"
+authors = ["TransitMatters Labs Team"]
+license = "MIT"
+
+[tool.poetry.dependencies]
+python = "~3.9"
+boxsdk = "3.7.2"
+numpy = "1.24.3"
+openpyxl = "3.1.2"
+pandas = "1.5.3"
+pytz = "2023.3"
+requests = "2.31.0"
+tqdm = "4.65.0"
+
+[tool.poetry.dev-dependencies]
+black = "23.3.0"
```

Comment:
For black, we have a config in other repos that we add to this file. It includes a longer line length and a target python version.

Also we use flake8 in other repos. Should we add that here as well?